dev/requirements_merge_arrow_pr.txt

```diff
@@ -1,3 +1,2 @@
 jira
```

Comment:
Is JIRA still supported?

Reply:
It is no longer supported as the issues where moved from jira to arrow gh issues: https://lists.apache.org/thread/01f570w32df5prr8tqwrsq5nq4vn570x

But we should probably wait with the removal until after 17.0.0 and remove all jira stuff from the release scritps as well or what do you think @raulcd

Reply:
I agree, better wait for 17.0.0 and remove all the JIRA related stuff both on the merge script, archery, etc